docs/source/building.rst

```diff
@@ -0,0 +1,112 @@
+.. _building:
+
+Building PYJNIus
+================
+
+Building PyJNIus depends on `Cython <http://cython.org/>`_ and the `Java
+<http://www.oracle.com/javase>`_ Development Kit (includes the Java Runtime
+Environment).
+
+
+Building on GNU/Linux distributions
+-----------------------------------
+
+You need the GNU Compiler Collection (GCC), the JDK installed (openjdk
+will do), and Cython. Then, just type::
+
+    pip install .
+
+If you want to compile the extension within the directory for any development,
+just type::
+
+    make
```

Comment:
Perhaps this is what requires Cython. pip install install it in the venv, right?

Reply:
This requires Cython, but we can change the make script (or delete it), since now things are a way more easier to do via `pip`.

Reply:
addressed by mentioning Cython here.